### ISSUE TITLE
Allow variables to pass property pattern rule

### ISSUE THREAD

**Issue by simPod** [stylelint/stylelint-config-standard#240]
### What is the problem you're trying to solve?

This is my rule

```scss
.a {
    --#{$variable-prefix}table-bg: rgba(#a1f2b0, 0.3);
}
```

that currently triggers `custom-property-pattern` rule error. Can we make it pass? 

(Not sure if this does not rather belong to https://github.com/stylelint-scss/stylelint-config-standard-scss)

### What solution would you like to see?

Allow that in `custom-property-pattern` rule

**Comment by jeddy3** [comment]
> Can we make it pass?

Yes, it's an easy fix.

However, please [open a bug report in the main stylelint repo](https://github.com/stylelint/stylelint/issues/new?assignees=&labels=&template=REPORT_A_BUG.yml) so that we can track the issue.

(I tried to transfer this issue, but GitHub is playing up.)
